pages/book/upgrades.mdx

```diff
@@ -1,3 +1,3 @@
 # Contracts upgrades
 
-Tact currently doesn't allow contract upgrades since Tact contracts have a more convoluted nature than the ones in `func`. It is theoretically possible, but the required tools are not here.
\ No newline at end of file
+Tact currently doesn't allow contract upgrades since Tact contracts have a more convoluted nature than the ones in `FunC`. It is theoretically possible, but the required tools are not here.
```

Comment:
```suggestion
Tact currently doesn't allow contract upgrades since Tact contracts have a more convoluted nature than the ones in FunC. It is theoretically possible, but the required tools are not here.
```